frontend-react/src/components/ReportStreamFooter.stories.tsx

```diff
@@ -1,3 +1,4 @@
+// AutoUpdateFileChromatic
```

Comment:
[non-blocking] Just a question - is this comment the signal to push to Chromatic? If so, what would be a reason to not push to Chromatic automagically?

Reply:
Yes, I can remove this comment but then this component will not push to chromatic automatically.